docs/source/tutorials/describing_tasks.ipynb

```diff
@@ -1,20 +1,563 @@
 {
```

Comment:
<p>I think about these tutorials more like a guide than an hands on tutorial. This is why I proposed to elminate the exercise and actually show users how to do the change. Would you agree?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Yeah, that's fine.</p>

Reply:
Done.